tests/workbook/serialization_test_suite.cpp

```diff
@@ -808,6 +809,21 @@ class serialization_test_suite : public test_suite
         wb.load(path_helper::test_file("20_active_sheet.xlsx"));
         xlnt_assert_equals(wb.active_sheet(), wb[2]);
     }
+
+    void test_locale_comma ()
+    {
+        struct SetLocale
+        {
+            SetLocale() {xlnt_assert(setlocale(LC_ALL, "de_DE") != nullptr);} // If failed, please install de_DE locale to correctly run this test.
```

Comment:
Same comment about setting and unsetting the global locale as above. This is fine for now, but we should improve it in the future :wink: